beluga/include/beluga/sensor/ndt_sensor_model.hpp

```diff
@@ -256,13 +256,19 @@ NDTMapRepresentationT load_from_hdf5_2d(const std::filesystem::path& path_to_hdf
   std::array<hsize_t, 2> dims_out;
   means_dataset.getSpace().getSimpleExtentDims(dims_out.data(), nullptr);
 
-  Eigen::Matrix2Xd means_matrix(dims_out[1], dims_out[0]);
-  Eigen::Matrix2Xi cells_matrix(dims_out[1], dims_out[0]);
+  // The static cast is necessary because hsize_t and size_t are not the same in all platforms.
+  std::array<std::size_t, 2> dims{
```

Comment:
@nahueespinosa nit: can be made `const`

Reply:
Done in https://github.com/Ekumen-OS/beluga/pull/392/commits/3b98df9263901ecb2b0535cfce30df3628657b0a.